site/css/_colors.css

```diff
@@ -19,6 +19,7 @@
 @value cainYellow: #FFD811;
 @value cainYellowFaded: #fede3f;
 @value scorpian: #606060;
+@value yoloPurple: #B88DFF;
```

Comment:
Potentially need to update this page with new colours http://brand.red-badger.com/d/WEl30uS4YzKw/red-badger-styleguide-style-guide#/elements/colours